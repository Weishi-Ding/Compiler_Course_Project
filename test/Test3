
/*test statement*/
int main(){
    /*while*/
    int i;
    while(i < 10){
        print(i);
        i = i + 2;
    }

    /*for*/
    for(i = 5; i < 10; i = i+1){
        print(i);
    }

    /*if*/
    if(i == i){
        print("is i");
    }

    /*if else*/
    if(i != i){

    }
    else{
        print("not i");
    }
}