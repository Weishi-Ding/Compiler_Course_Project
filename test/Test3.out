0
2
4
6
8
5
6
7
8
9
"is i"
"not i"
